app/src/store/playground/types.ts

```diff
@@ -138,7 +138,12 @@ interface PlaygroundInstanceActionParams {
   playgroundInstanceId: number;
 }
 
-export interface AddMessageParams extends PlaygroundInstanceActionParams {}
+export interface AddMessageParams extends PlaygroundInstanceActionParams {
+  /**
+   * If not provided, a default empty message will be added
+   */
+  messages?: ChatMessage | ChatMessage[];
```

Comment:
```suggestion
  messages?: ChatMessage[];
```
can we keep the type uniform?